Scripts/script-ChangeRemediationSLAOnSevChange.yml

```diff
@@ -0,0 +1,45 @@
+fromversion: 4.1.0
+commonfields:
+  id: changeremediationslaonsevchange
+  version: -1
+name: ChangeRemediationSLAOnSevChange
+script: |
+  import datetime
+
+  # ##### Help #####
+  # Triggered Field Name is in: demisto.args()['name']
+  # Field's old value is in: demisto.args()['old']
+  # Field's new value is in: demisto.args()['new']
+  # Printing argument structure: demisto.results(demisto.args())
+
+  # ##### Configuration #####
+  CRITICAL_SLA = 60  # In minutes
+  NONCRITICAL_SLA = 6 # In days
+
+  # ##### Logic #####
+  args_sev =  demisto.args()['new']
+  now = datetime.datetime.utcnow()
+
+  if args_sev == 'Critical':
+      demisto.log('Severity changed to Critical')
+      # newsla = now + datetime.timedelta(days=2)
+      demisto.executeCommand("setIncident",{'sla': CRITICAL_SLA, "slaField":"remediationsla"})
+
+  else:
+      demisto.log('Severity changed to Not Critical' )
+      newsla = now + datetime.timedelta(days=NONCRITICAL_SLA)
+      newsla = newsla.strftime('%Y-%m-%dT%H:%M:%S+00:00')
+      demisto.executeCommand("setIncident",{'sla': newsla, "slaField":"remediationsla"})
+type: python
+tags:
+- field-change-triggered
+- example
```

Comment:
I might be missing something here, but shouldn't we have args section? how are you getting the `old, new, name` args?

Reply:
This is on field change script , so those arguments would be injected from the server. It will work even without defining them. 
But I think we better define them, this way we will be able to test those scripts.